opal/mca/btl/sm/btl_sm_module.c

```diff
@@ -332,7 +333,6 @@ static int sm_add_procs (struct mca_btl_base_module_t* btl,
                             struct mca_btl_base_endpoint_t **peers,
                             opal_bitmap_t *reachability)
 {
-    mca_btl_sm_component_t *component = &mca_btl_sm_component;
```

Comment:
Unused variable warning?

Reply:
Yeah - I don't think these changes caused it to become unused, but it was simple enough to remove anyway.